src/linagora.esn.profile/app/app.module.js

```diff
@@ -1,3 +1,20 @@
+require('esn-frontend-common-libs/src/frontend/js/modules/core.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/config/config.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/esn.router.js');
+require('esn-frontend-common-libs/src/frontend/components/dynamic-directive/src/dynamic-directive.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/http.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user/user.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/profile.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/notification.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/timeline/timeline.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/previous-page.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/async-action.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/availability/availability.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/module-registry/module-registry.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user-configuration/user-configuration.module.js');
```

Comment:
You need to add the associated AngularJS dependency for user-configuration.module.js